playground/hmr/invalidation/child.js

```diff
@@ -0,0 +1,8 @@
+if (import.meta.hot) {
+  // Need to accept, to register a callback for HMR
+  import.meta.hot.accept()
+  // Trigger HMR in importers
+  import.meta.hot.invalidate()
```

Comment:
This code works because `import.meta.hot.invalidate()` is ignored for the first load by `mod.lastHMRTimestamp > 0`.

Though I suppose the code below better expresses what we want to do here.
```suggestion
  // Need to accept, to register a callback for HMR
  import.meta.hot.accept(() => {
    // Trigger HMR in importers
    import.meta.hot.invalidate()
  })
```

Reply:
Yeah that makes sense to me, and since the accept callback receives the new module instance, we can remove the dynamic import from #10239 (https://github.com/vitejs/vite/pull/10239/files#diff-97a53ff7c185281b1b608618378eb10aab2eb18ba01b0073b440621abc5914dfR103)

Reply:
Nice!